test/farm/001-farm.bats

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env bats
+#
+# Tests of podman farm commands
+#
+
+load helpers.bash
+
+###############################################################################
+# BEGIN tests
+
+fname="test-farm"
+containerfile="test/farm/Containerfile"
+
+@test "farm - check farm has been created" {
+    run_podman farm ls
+    assert "$output" =~ $fname
+    assert "$output" =~ "test-node"
+}
+
+@test "farm - build on local only" {
+    iname="test-image-1"
+    empty_farm="empty-farm"
+    # create an empty farm
+    run_podman farm create $empty_farm
+    run_podman farm --farm $empty_farm build -f $containerfile -t $iname .
+    assert "$output" =~ "Local builder ready"
+    # get the system architecture
+    getArch
+    # inspect manifest list built and saved in local containers-storage
+    run_podman manifest inspect $iname
+    assert "$output" =~ $ARCH
+}
+
+@test "farm - build on farm node only with --cleanup" {
+    iname="test-image-2"
+    run_podman farm build -f $containerfile --cleanup --local=false -t $iname .
+    assert "$output" =~ "Farm \"$fname\" ready"
+    # get the system architecture
+    getArch
+    # inspect manifest list built and saved in dir
+    manifest=$(cat $iname/manifest.json)
```

Comment:
I'm having trouble understanding where this file comes from?

Reply:
This gets created when we build on the remote node only and pull the image. It gets saved in a dir using the dir transport.